### DISCUSSION TITLE
Next.js + Emotion + evergreen-ui

### DISCUSSION THREAD

**Discussion by kud** [vercel/next.js#12619] [Help] [answered]
Hello,

I'm currently doing my blog, using next.js + emotion, which work pretty well since emotion 10.

However, when I use evergreen-ui (a UI Kit), it works well when developing but not when exporting:

- dev

![image](https://user-images.githubusercontent.com/655838/81354381-b9e86f00-90cb-11ea-8f73-4318a8d1778e.png)

- dist

![image](https://user-images.githubusercontent.com/655838/81354395-c371d700-90cb-11ea-9bdc-554f76b48994.png)

And the reason is because when you've got the exported version, you don't have the evergreen-ui css included:

![image](https://user-images.githubusercontent.com/655838/81354437-e603f000-90cb-11ea-915e-de25d534c8cf.png)

Do you have any idea why?

Thank you.

**Comment by jamesmosier** [comment, marked as answer]
How and where are you importing evergreen?

**Reply by kud** [reply]
In a layout or a page, via 

```jsx
import { HomeIcon } from "evergreen-ui"

const MyIcon = styled(HomeIcon)``
```

**Reply by jamesmosier** [reply]
I was able to get it to work like this:

```js
import { HomeIcon, setClassNamePrefix } from "evergreen-ui";
setClassNamePrefix("ub-");
```

It appears there are issues with their current className prefix of a box emoji and Next, although I'm not certain without digging in further.

**Reply by kud** [reply]
Indeed, they're having fun with the prefix of the class names.

Thank you! I'll try this!
